pkg/services/ngalert/api/generated_base_api_provisioning.go

```diff
@@ -101,121 +101,121 @@ func (f *ForkedProvisioningApi) RoutePutTemplate(ctx *models.ReqContext) respons
 func (api *API) RegisterProvisioningApiEndpoints(srv ProvisioningApiForkingService, m *metrics.API) {
 	api.RouteRegister.Group("", func(group routing.RouteRegister) {
 		group.Delete(
-			toMacaronPath("/api/provisioning/contact-points/{ID}"),
-			api.authorize(http.MethodDelete, "/api/provisioning/contact-points/{ID}"),
+			toMacaronPath("/api/provisioning/v1/contact-points/{ID}"),
```

Comment:
Do other provisioning endpoints similarly have a version segment in their paths? I think it's a great idea, and we should promote it for other APIs.

Reply:
Nothing in the Grafana repo uses API versioning _at all_ right now. We raised this in a few discussion sessions during Grafanafest, and generally got good feedback from engineers on other teams.

I made a design doc explaining the rationale (this PR is draft, while I finish that up). That way, we have a record of the decision that we can link around and share in wider audiences.

Reply:
awesome, love it.